classes.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Classes</title>
+    <link rel="stylesheet" a href="classes.css">
+  </head>
+  <body>
+      <div class="nav">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="story.html">Story</a></li>
+          <li><a href="classes.html">Classes</a></li>
+          <li><a href="world.html">World</a></li>
+        </ul>
+      </div>
+    <h1>Heroes of Sanctuary</h1>
+    <h2 style="text-align: center; color: white; padding: 5rem;">Five classes have been announced. The Barbarian, Sorceress, and Druid classes were announced at BlizzCon 2019, while a fourth, the Rogue class, was announced at BlizzCon 2021. The fifth class, the Necromancer, was announced in 2022.</h2>
```

Comment:
Ai linii de cod foarte lungi. Poate le mai spargi. Încearcă să te limitezi pe la 100 de caractere pe o linie.

```suggestion
    <h2 style="text-align: center; color: white; padding: 5rem;">
      Five classes have been announced. The Barbarian, Sorceress, and Druid classes were announced 
      at BlizzCon 2019, while a fourth, the Rogue class, was announced at BlizzCon 2021. The fifth 
      class, the Necromancer, was announced in 2022.
    </h2>
```

Reply:
Am modificat acum.